kotlin-analysis-api/src/main/kotlin/com/google/devtools/ksp/impl/symbol/kotlin/resolved/KSTypeReferenceResolvedImpl.kt

```diff
@@ -66,7 +67,13 @@ class KSTypeReferenceResolvedImpl private constructor(
                 ktType is KtClassErrorType || (ktType.classifierSymbol() == null)
             }
         ) {
-            KSErrorType
+            KSErrorType(
+                when (ktType) {
+                    is KtClassErrorType -> ktType.getNameHint()
+                    is KtTypeErrorType -> null // No info available
```

Comment:
we can look at the parent for possible information, you can ignore this comment if it is too much work for you.

Reply:
I've noticed that when there's some type information (at least an identifier, or even identifier matched with an import/qualified identifier) then we get a `KtClassErrorType` which is good. When the type is simply missing altogether - we get `KtTypeErrorType`. Nevertheless I've used `KtTypeErrorType.tryRenderAsNonErrorType` as a last resort, as some info may still be available though there. The place where we handle AA's error types is now `KtType.render()` function for consistency.